stake-pool/program/src/instruction.rs

```diff
@@ -41,7 +42,7 @@ pub enum StakePoolInstruction {
     ///   8. `[]` Rent sysvar
     ///   9. `[]` Token program id
     Initialize {
-        /// Deposit fee assessed
+        /// Fee assessed as percentage of perceived rewards
         #[allow(dead_code)] // but it's not
         fee: Fee,
```

Comment:
Totally fine as a follow-up, but I think we'll want an instruction to allow the fee to be changed after initialization

Reply:
Yeah definitely, I didn't want to do put too much into this PR